test/other/test_unoptimized_code_size_strict.js.size

```diff
@@ -1 +1 @@
-57018
+58622
```

Comment:
These are unoptimized sizes so I guess its OK that they get larger... but I'm kind of surprised we don't see any code size wins for the optimized builds..  any idea why we don't?

Reply:
They are not, as mentioned in description this initially looked like #20530 but actually turned out to be https://github.com/emscripten-core/emscripten/pull/20578 (see that other PR, I'll rebaseline when that fix is in).

Reply:
Great